test/main.spec.tsx

```diff
@@ -1,14 +1,12 @@
-
+/* eslint-disable jest/expect-expect */
 /* eslint-disable @typescript-eslint/no-explicit-any */
-import React from 'react';
-import { MainStack } from '../src/main';
-import renderer, { act } from 'react-test-renderer';
+import { act } from 'react-dom/test-utils';
+import { renderMain } from '../src/main';
 
 describe('Main', () => {
   it('renders correctly', () => {
     act(() => {
-      const mainStack = renderer.create(<MainStack/>).toJSON();
-      expect(mainStack).toMatchSnapshot();
```

Comment:
@logrdog every unit test has an expect statement so this is not correct here

Reply:
@JoshuaVSherman Sorry, I assumed it was good because it hit the function on coverage. I added the expect.